### PR TITLE
Improvements on some widgets

### PR DESCRIPTION
**Pull request by fperezluna**
spinner component is added

### CODE REVIEW

File: src/widgets/spinner.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+	public async increase() {
+		await this.elem.element(by.className('input-group-append')).click();
+	}
+
+	public async decrease() {
+		await this.elem.element(by.className('input-group-prepend')).click();
+	}
+
+	public async isEnabled(): Promise<boolean> {
```
Comment (XavierCalvo) [inline src/widgets/spinner.ts]: Why do you override isEnable? It is already in Widget class

File: src/widgets/spinner.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import {by} from "protractor";
+import {Widget} from "./widget";
+
+export class Spinner extends Widget {
+
+	public async clear() {
```
Comment (alfonsserra) [inline src/widgets/spinner.ts]: Add :void as return type

File: src/widgets/spinner.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+	public async setText(text: string): Promise<void> {
+		await this.elem.element(by.tagName('input')).sendKeys(text);
+	}
+
+	public async getText(): Promise<string> {
+		return await this.elem.element(by.tagName('input')).getAttribute('value');
+	}
+
+	public async increase() {
```
Comment (alfonsserra) [inline src/widgets/spinner.ts]: add : Promise<void>  as return type

File: src/widgets/spinner.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+		await this.elem.element(by.className('input-group-append')).click();
+	}
+
+	public async decrease() {
```
Comment (alfonsserra) [inline src/widgets/spinner.ts]: add : Promise<void>  as return type

File: src/index.ts [hunk 5]
```diff
@@ -7,7 +7,9 @@ export * from './widgets/icon';
 export * from './widgets/inputfield';
 export * from './widgets/label';
 export * from './widgets/message-popup';
+export * from './widgets/multiple-combobox';
```
Comment (alfonsserra) [inline src/index.ts]: Have you added multiple-combobox.ts file?

File: src/widgets/spinner.ts (hunk 1 above)
Comment (fperezluna) [reply src/widgets/spinner.ts]: function removed

File: src/index.ts (hunk 5 above)
Comment (fperezluna) [reply src/index.ts]: multiple-combobox was already added in the previous pull request but it was not included in index

File: src/widgets/spinner.ts (hunk 1 above)
Comment (fperezluna) [reply src/widgets/spinner.ts]: function removed

File: src/widgets/spinner.ts (hunk 2 above)
Comment (fperezluna) [reply src/widgets/spinner.ts]: type added

File: src/widgets/spinner.ts (hunk 3 above)
Comment (fperezluna) [reply src/widgets/spinner.ts]: promise added

File: src/widgets/spinner.ts (hunk 4 above)
Comment (fperezluna) [reply src/widgets/spinner.ts]: promise added

**Pull request merged**